rest-api/src/main/resources/paths/participants/v3_participants_userId_reports_identifier.yml

```diff
@@ -19,5 +19,28 @@ get:
                 $ref: ../../definitions/paged_resources/report_data.yml
         401:
             $ref: ../../responses/401.yml
+post:
+    operationId: addParticipantReportRecord
+    summary: Add a participant report record
+    description: The report can be accessed if the participant is requesting their own report, or the caller is a researcher, worker, or admin, or the caller is a study coordinator for a study that includes the participant.
+    tags:
+        - Participant Reports
+        - _For Developers
+    security:
+        - BridgeSecurity: []
+    parameters:
+        - $ref: ../../index.yml#/parameters/userId
+        - $ref: ../../index.yml#/parameters/identifier
+        - name: ReportData
+          required: true
+          in: body
+          schema:
+            $ref: ../../definitions/report_data.yml
+
+    responses:
+        201:
+            $ref: ../../responses/201_message.yml
+        401:
+            $ref: ../../responses/401.yml
         403:
-            $ref: ../../responses/403_not_researcher.yml
+            $ref: ../../responses/403_not_developer_researcher.yml
```

Comment:
Shouldn't this also allow study coordinator to add a report?

Reply:
Yes, added here and in the v4 version of the same API (both endpoints are mapped to the same controller endpoint).